### ISSUE TITLE
Optimize Jackson extension performance

### ISSUE THREAD

**Issue by jlouvel** [restlet/restlet-framework-java#715]
We should facilitate the reuse of expensive objects such as ObjectMapper.

See this discussion thread:
http://restlet.tigris.org/ds/viewMessage.do?dsForumId=4447&dsMessageId=3044375

Some references:
http://wiki.fasterxml.com/JacksonBestPracticesPerformance
https://github.com/Tembrel/restlet-misc/blob/master/src/main/java/net/peierls/restlet/misc/LocalJacksonConverter.java

**Comment by qsiebers** [comment]
For those running into this:

My current solution is to create a `ConverterService` subclass that calls `JacksonRepresentation.setObjectMapper`, `JacksonRepresentation.setObjectReader` and `JacksonRepresentation.setObjectWriter` with pre-configured (reused) versions.

I think the main hurdle to solve this in the restlet code base is that the `JacksonRepresentation.createObjectMapper` method create a different `ObjectMapper` per `MediaType`, which makes it hard to reuse it.

**Comment by qsiebers** [comment]
@Tembrel Yes, I've applied that approach before [here](https://github.com/ontopia/ontopia/blob/master/ontopia-rest/src/main/java/net/ontopia/topicmaps/rest/converters/jackson/JacksonConverterImpl.java).
